docs/source/en/installation.md

```diff
@@ -82,11 +51,11 @@ Run the command below to check if your system detects an NVIDIA GPU.
 nvidia-smi
 ```
 
-To install a CPU-only version of Transformers and a machine learning framework, run the following command.
+To install a CPU-only version of Transformers, run the following command.
 
 ```bash
-pip install 'transformers[torch]'
-uv pip install 'transformers[torch]'
+uv pip install torch --index-url https://download.pytorch.org/whl/cpu
```

Comment:
Do we need uv?

uv is an extra dependency, only recently being noticed by https://www.pypa.io https://packaging.python.org/en/latest/tutorials/managing-dependencies/ 

(Addition of uv was requested in https://github.com/astral-sh/uv/issues/14672)